security/android_vbmeta.ksy

```diff
@@ -0,0 +1,322 @@
+          "avbtool 1.0.0 xyz_board Git-234abde89". Is guaranteed to be NUL
+          terminated. Applications must not make assumptions about how this
+          string is formatted.
+      - id: reserved
+        size: 80
+        doc: |
+          Padding to ensure struct is size AVB_VBMETA_IMAGE_HEADER_SIZE
+          bytes. This must be set to zeroes.
+    instances:
+      is_hashtree_disabled:
+        value: flags & 1 != 0 
+      is_verification_disabled:
+        value: flags & 2 != 0
+  vbmeta_public_key_header:
+    doc: The header for a serialized RSA public key
+    seq:
+      - id: key_num_bits
+        type: u4
+        doc: The size of the key in bits
+      - id: n0inv
+        type: u4
+        doc: Precomputed value for optimization of verification.
+  property_descriptor:
+    doc: A descriptor for properties (free-form key/value pairs)
+    seq:
+      - id: key_num_bytes
+        type: u8
+      - id: value_num_bytes
+        type: u8
+      - id: key
+        size: key_num_bytes
+      - id: stub1
+        contents: [0]
+      - id: value
+        size: value_num_bytes
+      - id: stub2
+        contents: [0]
+  hashtree_descriptor:
+    doc: A descriptor containing information about a dm-verity hashtree.
+    seq:
+      - id: dm_verity_version
+        type: u4
+      - id: image_size
+        type: u8
+      - id: tree_offset
+        type: u8
+      - id: tree_size
+        type: u8
+      - id: data_block_size
+        type: u4
+      - id: hash_block_size
+        type: u4
+      - id: fec_num_roots
+        type: u4
+      - id: fec_offset
+        type: u8
+      - id: fec_size
+        type: u8
+      - id: hash_algorithm
+        size: 32
+      - id: partition_name_len
+        type: u4
+      - id: salt_len
+        type: u4
+      - id: root_digest_len
+        type: u4
+      - id: flags
+        type: u4
+        enum: descriptor_hashtree_flags
+      - id: reserved
+        size: 60
+      - id: partition_name
+        type: str
+        size: partition_name_len
+        encoding: utf-8
+      - id: salt
+        size: salt_len
+      - id: root_digest
+        size: root_digest_len
+  hash_descriptor:
+    doc: A descriptor containing information about hash for an image.
+    seq:
+      - id: image_size
+        type: u8
+      - id: hash
+        size: 32
+      - id: partition_name_len
+        type: u4
+      - id: salt_len
+        type: u4
+      - id: digest_len
+        type: u4
+      - id: flags
+        type: u4
+        enum: descriptor_hash_flags
+      - id: reserved
+        size: 60
+      - id: partition_name
+        type: str
+        size: partition_name_len
+        encoding: utf-8
+      - id: salt
+        size: salt_len
+      - id: digest
+        size: digest_len
+  kernel_cmdline_descriptor:
+    doc: A descriptor containing information to be appended to the kernel command-line.
+    seq:
+      - id: flags
+        type: u4
```

Comment:
Should it be a type of bit fields (`b1` for single bit, `b6` for 6 bits) instead of enummed  `u4`?